mantidproject/mantid
# DGS reduction interface bug

Fix #11602 

Changed how the parameters are handled, from int to string. This allows for empty  inputs for the monitor spectra. Also put a try...except around the UI checking when one clicks the reduce button

To test: 
1. Open DgsReduction interface (not the algorithm), choose your facility and instrument, choose a file. By default, the GUI is populated with the default monitor spectra numbers. See that you can reduce the file.
2. Delete the content for spectrum numbers for monitor 1 and 2. re-run the reduction. Before these changes, the script will produce an error. When the reduction button is clicked, all buttons at the bottom are disabled, until the reduction finishes. If the UI fails, that exception is not caught, and the buttons are not re-enabled. This should not happen with this fix

Release notes: None - minor bugfix
